legend-engine-language-pure-dsl-service/src/main/java/org/finos/legend/engine/language/pure/dsl/service/grammar/from/ServiceParseTreeWalker.java

```diff
@@ -103,6 +107,14 @@ public Service visitService(ServiceParserGrammar.ServiceContext ctx)
         return service;
     }
 
+    private ServiceTag visitServiceTagDescription(ServiceParserGrammar.TagDescriptionContext ctx)
+    {
+        ServiceTag serviceTag = new ServiceTag();
+        serviceTag.name = ctx.serviceTagName().STRING().getText();
+        serviceTag.value = ctx.serviceTagValue().STRING().getText();
+        return serviceTag;
+    }
```

Comment:
Are tags validated in any way in compilation?
If so, we should have a sourceInformaiton here and also add tests with a compilation failure exposing the error in the correct line/column

Reply:
The tags aren't validated in compilation as these are just trivial strings, so compilation check won't be necessary